src/PhpSpec/CodeGenerator/Generator/ClassGenerator.php

```diff
@@ -54,7 +54,7 @@ protected function renderTemplate(ResourceInterface $resource, $filepath)
             '%name%'            => $resource->getName(),
             '%namespace%'       => $resource->getSrcNamespace(),
             '%namespace_block%' => '' !== $resource->getSrcNamespace()
-                                ?  sprintf("\n\nnamespace %s;", $resource->getSrcNamespace())
+                                ?  sprintf("%snamespace %s;", PHP_EOL.PHP_EOL, $resource->getSrcNamespace())
```

Comment:
I agree with @stof. There's no way we should be using the OS specific newline character.